trans-calc.py

```diff
@@ -67,6 +68,9 @@ def create_ui_grid(self) -> None:
         self.edit_client_button.grid(column=1, row=0, padx=(5, 5))
         self.delete_client_button.grid(column=2, row=0, padx=(5, 5))
 
+        self.clear_button = ttk.Button(self.secondary_frame, command=self.clear_fields, text="Clear Fields")
```

Comment:
Please explain this line, as the `clear_button` is already defined at line 50. Atm, duplicate code.

Reply:
I am just blind idk how i missed it xd